fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersionsIT.java

```diff
@@ -264,7 +264,7 @@ public void testCreateTwoVersionsWithSameLabel() throws Exception {
                 new HttpGet(serverAddress + objId + "/fcr:versions");
         logger.debug("Retrieved versions");
         final GraphStore results = getGraphStore(getVersions);
-        assertEquals("Expected exactly 3 triples!", 3, countTriples(results));
+        assertEquals("Expected exactly 3 triples!", 5, countTriples(results));
```

Comment:
I do not expect this PR necessarily go through, as creating version checkpoints when adding the duplicate label would cause the operation to fail does not seem right... but if this PR did go through, yes, the assertion text should be updated to "Expected exactly 5 triples!"

Thoughts? I have also asked for @mikedurbin's feedback.